src/store/selectors/isGroupRoute.js

```diff
@@ -10,7 +10,7 @@ const isGroupRoute = createSelector(
 export default isGroupRoute
 
 export function getSlugFromLocation (state, props) {
-  return getRouteParam('groupSlug', state, props, false) || tryLocation(props) || props.groupSlug
+  return getRouteParam('groupSlug', state, props, false) || getRouteParam('detailGroupSlug', state, props, false) || tryLocation(props) || props.groupSlug
```

Comment:
this makes me nervous, is a detailGroup the same as a group route? are we sure this doesnt cause issues in some other cases of checking isGroupRoute?

Reply:
getGroupForDetail seems like a different case. we should really comment more on these things